mlflow/protos/databricks_managed_catalog_messages.proto

```diff
@@ -52,3 +52,83 @@ message GetTableResponse {
   // [Create,Update:IGN] Id of the table.
   optional string table_id = 22;
 }
+
```

Comment:
We probably should move these outside of `databricks_managed_catalog.messages` and into something new.  maybe `unity_catalog_oss.messages`